README.CentOS7.md

```diff
@@ -12,16 +11,17 @@ For further details on the install process, please see the LORIS GitHub Wiki Cen
 
 # System Requirements
 
-Default dependencies installed by CentOS 6.x may not meet the version requirements LORIS deployment or development.
-* MySQL 5.7 is supported for LORIS 18.*
-* PHP 7 is supported for LORIS 18.* - upgrade your PHP manually
+Default dependencies installed by CentOS 7.x may not meet the version requirements LORIS deployment or development.
+* MySQL 5.7 is supported for LORIS 22.*
+* PHP 7.2 is supported for LORIS 22.* - upgrade your PHP manually
```

Comment:
didn't knew that Loris is already at V22!
Should be Loris 21.*

Reply:
it's on the `major` branch so by definition v22. (This readme will need this update eventually anyway)
Shen issued a similar PR to `bugfix` for release 21.*

I'm ok with this being 21 for now though - thoughts? @PapillonMcGill

Reply:
Haven't realized it was for major. That explain why there are 2 almost identical PRs.
In that case, no objection.